docs/generated/packages/jest/documents/overview.md

```diff
@@ -96,6 +96,18 @@ This is because each [jest process creates a workers based on system resources](
 
 Primary configurations for Jest will be via the `jest.config.ts` file that generated for your project. This file will extend the root `jest.preset.js` file. Learn more about [Jest configurations](https://jestjs.io/docs/configuration#options).
 
+The root level `jest.config.ts` file configures [Jest multi project support](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig). This is allows having the Jest CLI and editor/IDE integrations working without Nx. Nx uses a utility function called `getJestProjects` which collects a list of Jest project configurations in the workspace instead of statically defining a list in the configuration file.
```

Comment:
The wording seems a bit awkward: `This is allows having the Jest CLI and editor/IDE integrations working without Nx`

Maybe something like `This root configuration file is allows Jest CLI and editor/IDE integrations to work without Nx`?